scripts/README.md

```
@@ -0,0 +1,283 @@
+- パフォーマンス設定
+- ファイルシステムの構造
+- 依存関係のインストール状況
+
+**実行方法**:
+```bash
+tsx scripts/validate-configuration.ts
+```
+
+**出力例**:
+```
+================================================================================
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)